tests/packerlicious/test_examples.py

```diff
@@ -1,70 +1,35 @@
 # Copyright (c) 2012-2013, Mark Peek <mark@peek.org>
 # All rights reserved.
-#
-from __future__ import unicode_literals
 
 import os
 import re
-import sys
-import unittest
-import pytest
-from io import StringIO
+from importlib import import_module
 
-try:
-    u = unicode
-except NameError:
-    u = str
 
-@pytest.mark.skip
-class TestExamples(unittest.TestCase):
+class TestFileExamples(object):
 
-    maxDiff = None
+    output_files = None
 
-    # those are set by create_test_class
-    filename = None
-    expected_output = None
+    def load_output_files(self):
+        self.output_files = {}
+        regex = re.compile(r'.template', re.I)
+        current_directory = os.path.dirname(os.path.realpath(__file__)) + '/../' + 'examples_output'
+        example_filenames = filter(regex.search, os.listdir(current_directory))
+        for example_filename in example_filenames:
+            with open(current_directory + '/' + example_filename) as f:
+                self.output_files[example_filename] = f.read()
 
-    def test_example(self):
-        saved = sys.stdout
-        stdout = StringIO()
-        try:
-            sys.stdout = stdout
+    def load_example_file(self, example_output_filename):
+        original_example = example_output_filename.split('.')[0]
+        example_module = import_module('examples.' + original_example)
+        assert self.output_files[original_example + '.template'].rstrip() == example_module.t.to_json()
```

Comment:
if there is multiple example files, how will you be able to tell which one fails?

Reply:
I just added a message for the assert.